Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -92,6 +92,8 @@
     if (outError)
         *outError = nil;
 
+    Ref<WebExtensionContext> protectedExtensionContext = extensionContext;
```

Comment:
Should be able to use just `Ref` here without a template argument because of deduction guides.